src/Framework/TestResult.php

```diff
@@ -578,6 +598,15 @@ public function passed(): array
         return $this->passed;
     }
 
+    /**
+     * Returns the names of the TestSuites that have passed.
+     * This enables @depends-annotations for TestClasName::class
+     */
+    public function passedClasses(): array
```

Comment:
```suggestion
    public function passedTestClasses(): array
```

What do you think?

Reply:
I like it better, it's more descriptive. Fixed.